samples/spring-security-xsuaa-usage/approuter/xs-app.json

```diff
@@ -1,7 +1,15 @@
 {
-  "routes": [{
-    "source": "^/",
-    "target": "/hello-token",
-    "destination": "token-destination"
-  }]
+  "routes": [
+    {
+      "source": "^/v1",
+      "target": "/v1",
+      "csrfProtection": false,
```

Comment:
what is the reason to disable csrf protection?

Reply:
for status changing requests, e.g. POST this would require an additional CSRF token. "To obtain a CSRF token one must send a GET or HEAD request with a x-csrf-token: fetch header to the application router. The application router will return the created token in a x-csrf-token: <token> header, where <token> will be the value of the CSRF token." This is not in focus of the sample and that's why i've disabled it to simplify testing with Postman REST client.

Reply:
removed csrfProtection flag and removed POST-endpoint